daemon/execdriver/driver.go

```diff
@@ -72,22 +78,33 @@ type Network struct {
 	HostNetworking bool              `json:"host_networking"`
 }
 
-// IPC settings of the container
+// Ipc settings of the container
+// It's for IPC namespace setting, usually different containers
+// have their own IPC namespaces, but we can specify a certain
+// IPC namespace for the container to join, it could belong to
+// a container or to the host.
 type Ipc struct {
 	ContainerID string `json:"container_id"` // id of the container to join ipc.
 	HostIpc     bool   `json:"host_ipc"`
 }
 
-// PID settings of the container
+// Pid settings of the container
+// It's for PID namespace setting, usually different containers
+// have their own PID namespaces, but we can specify a certain
+// PID namespace for the container to join, now we only support
```

Comment:
Just a grammar fixup:

```
It is for the PID namespace setting. Usually different containers
have their own PID namespace, however this specifies to use
an existing PID namespace.
Joining the host's PID namespace is currently the only supported option
```

You can this as a template for the IPC doc as well.